### ISSUE TITLE
Cannot upgrade from 5.5.09 to 5.6.0.3

### ISSUE THREAD

**Issue by jfwm2** [aerospike/aerospike-kubernetes-operator#44]
When trying to upgrade from 5.5.09 to 5.6.0.3 using last release (1.0.1) I obtain the following error:
```
Error: UPGRADE FAILED: cannot patch "aerocluster" with kind AerospikeCluster: admission webhook "aerospikeclusters.aerospike.com" denied the request: AerospikeConfig not valid: Generated config not valid for version 5.6.0.3: []
```
Is it a known issue?

**Comment by sud82** [comment]
Hi @jfwm2, 5.6.0.3 is not supported yet. We will add a better error message for these failures.

**Comment by jfwm2** [comment]
Thanks @sud82 
When do you expect it to be supported. If you could give me few pointers I may do a PR.

**Comment by sagesheryl** [comment]
Fixed in the 2.0 release. Tracking under PROD-1877 Add support for 5.6 and rework the error message to provide a more detailed error message for unsupported versions.

**Comment by sud82** [comment]
This has been fixed in master
